src/env.h

```diff
@@ -255,6 +255,7 @@ class Environment {
     inline uint32_t* fields();
     inline int fields_count() const;
     inline bool call_init_hook();
+    bool enabled;
```

Comment:
I have exposed it though the `fields_` array as you sugested. Is this something there should be tested? It looks like async-wrap have a different testing policy.